tsconfig.base.json

```diff
@@ -351,6 +351,9 @@
       "@energinet-datahub/watt/danish-date-time": [
         "libs/watt/src/lib/configuration/watt-danish-datetime.providers.ts"
       ],
+      "@energinet-datahub/watt/data": [
```

Comment:
Consideration: Of course, `data` is an excellent short import. However, does it make sense for everyone that this is the data-table entry point?

Reply:
I'm probably getting ahead of myself here, but I was imagining the `data` category to have other composite components for data presentation, such as a `watt-data-calendar` for example. But I do agree that the naming is a bit vague.

We could make a separate `@energinet-datahub/watt/data-table` entry point, but what about the imports that are not strictly for the table? For example (and again, getting ahead of myself 😄) the `watt-data-calendar` would also need the `watt-data-filters` as it has options for filtering what is shown on the calendar.

Do you have any suggestions?

Reply:
I buy in on that plan. Let's keep it.